ground/src/main/java/com/google/android/ground/ui/syncstatus/SyncStatusDetail.kt

```diff
@@ -17,10 +17,16 @@ package com.google.android.ground.ui.syncstatus
 
 import com.google.android.ground.model.mutation.Mutation
 
-/** A tiny helper class for bundling mutation history display data. */
-data class MutationDetail(
+/**
+ * Defines the set of data needed to display the human-readable status of a queued local [Mutation].
+ */
+data class SyncStatusDetail(
+  /** The username of the user who made this change. */
   val user: String,
+  /** The underlying [Mutation]. */
```

Comment:
Will we replace this with `UploadStatus` in a follow-up?

Reply:
sgtm, will do!